client/state/selectors/get-reader-teams.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns all of the reader teams for a user
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array}          Reader Teams
+ */
+export default function getReaderTeams( state ) {
+	return get( state, [ 'reader', 'teams', 'items' ] );
```

Comment:
no need for get, all the items in the path are guaranteed to exist by the reducer